### DISCUSSION TITLE
Can I exit yazi to the directory I navigated to?

### DISCUSSION THREAD

**Discussion by benekreng** [sxyazi/yazi#637] [1. Q&A]
In ranger, after I navigated somewhere, I can press `shift + s` to quit ranger and it automatically cd's into the directory where I was in ranger.

**Comment by SolitudeSF** [comment]
https://yazi-rs.github.io/docs/quick-start#shell-wrapper

**Reply by SolitudeSF** [reply]
also, `shift+s` in ranger doesnt quit ranger, it launches new shell process that inherits ranger's cwd. so you can get a nested chain of ranger+shell is you use that.

**Reply by benekreng** [reply]
Ahh thanks a lot! I totally forgot thats what `shift + s` does. I began using it for convenience cause I didn't have to decide beforehand wether I want to exit to the new directory. `cd -` will do the job too... 😄

**Reply by sxyazi** [reply]
This tip sounds like what you're describing, https://yazi-rs.github.io/docs/tips#dropping-to-the-shell
